### PR TITLE
Sequence motion planning

### PR DESCRIPTION
**Pull request by Flova**

### CODE REVIEW

File: marimbabot_planning/src/move_to_pose.cpp [hunk 1]
```diff
@@ -278,6 +282,7 @@ int main(int argc, char **argv)
     // Hit a sequence of points in cartesian space left and right of "pose"
 
     // Define hit points
+<<<<<<< Updated upstream
```
Comment (Flova) [inline marimbabot_planning/src/move_to_pose.cpp:285]: It seems like you pushed an unresolved merge

Comment (Flova) [reply marimbabot_planning/src/move_to_pose.cpp:285]: This is also the reason for the CI failure.

File: marimbabot_planning/src/move_to_pose.cpp [hunk 2]
```diff
@@ -24,15 +24,16 @@ moveit::planning_interface::MoveGroupInterface::Plan concatinated_plan(std::vect
     assert(plans.size() > 0);
 
     moveit::planning_interface::MoveGroupInterface::Plan plan{plans[0]};
-    ros::Duration time_from_start{0};
+    ros::Duration time_from_start{1};
```
Comment (Flova) [inline marimbabot_planning/src/move_to_pose.cpp:27]: Why did you do this? @berkgungor semantically zero is correct. Also, what is `time_added`?

**Review by berkgungor — APPROVED** [review]
LGTM!!!

**Pull request merged**